composeml/label_times/description.py

```diff
@@ -20,11 +20,9 @@ def describe_label_times(label_times):
     metadata = label_times.settings
     target_column = metadata["label_times"]["target_columns"][0]
     target_type = metadata["label_times"]["target_types"][target_column]
-    target_dataframe_name = metadata["label_times"]["target_dataframe_name"]
```

Comment:
I am unsure about what this does and if we should remove this or not.

Reply:
When I look at the `LabelTimes` class definition, it seems like we have some pretty confusing terminology. I think in one sense `target_column_name` refers to the column we want to use as the "target" for creating labels (as in, I want to create labels for my customers using `customer_id`).  

Then `target_column` here seems to refer to the column that is the machine learning target column (the label column, as in, `total_spent` for each customer).

I'm not sure how best to resolve this off the top of my head, but we should probably only use target in one case.

Reply:
@jeff-hernandez Any idea what to do here?

Reply:
I think the intent is to use the same terminology as featuretools. For example, in the past, we used `target_entity` to specify which entity to create features for. Similarly, the idea was the use `target_entity` to specify which entity to create labels for. The reason we put it in the metadata is so that we have the settings for reconstructing the label times.

Reply:
Maybe using `target_column_index` instead of `target_column_name` can help clarify